BaseFilter/sections/general_extensions.txt

```diff
@@ -2154,7 +2154,7 @@ mr-jatt.in#%#//scriptlet("json-prune", "rot_url pop_type")
 mr-jatt.in#%#//scriptlet("prevent-addEventListener", "", "x20click")
 ! https://github.com/AdguardTeam/AdguardFilters/issues/59478
 ! popunder when click on the back button
-vrsumo.com#%#Object.defineProperty(window, 'offer', { value: ''});
+vrsumo.com#%#//scriptlet('set-constant', 'offer', '')
```

Comment:
I'll revert this and open an issue in the scriptlets.
When you open this link and click on the back button on mouse, you get redirected. The scriptlet does not work, but the js rule does.

Reply:
Hmm. I checked it and was no redirects.

Reply:
I've found another scriptlet that works fine, replaced the js rule.